.github/ISSUE_TEMPLATE.md

```diff
@@ -1,50 +1,42 @@
 ## Classification:
-
-(Pick one of: Bug, Crash/Hang/Data Loss, Performance, UI/Usability, Feature (New), Enhancement)
+(Pick one: Bug, Crash/Hang/Data loss, Enhancement, Feature (new), Performance, UI/Usability)
```

Comment:
Most markdown flavours won't render headings without a newline after them. Could you verify that github will render something like this correctly?

## My test
No newline

Reply:
Hmm, github seems fine with it.

Reply:
Yep. You can see the proposed template in action [here](https://github.com/autokey/autokey/blob/c32fb4c59b890288ec6c731aea3a97d27139d826/.github/ISSUE_TEMPLATE.md
).